51.md

```diff
@@ -80,6 +82,23 @@ Some clients have used these lists in the past, but they should work on transiti
 }
 ```
 
+### A _mute list_ with the items expiring on January 1st 2025 00:00 UTC
+
+```json
+{
+  "id": "a92a316b75e44cfdc19986c634049158d4206fcc0b7b9c7ccbcdabe28beebcd0",
+  "pubkey": "854043ae8f1f97430ca8c1f1a090bdde6488bd5115c7a45307a2a212750ae4cb",
+  "created_at": 1699597889,
+  "kind": 10000,
+  "tags": [
+    ["p", "07caba282f76441955b695551c3c5c742e5b9202a3784780f8086fdcdc1da3a9", "1735689600"],
```

Comment:
The second index in a "p" tag should always be the relay hint. so id say you could either move the timestamp to the third index or maybe it might make sense to use an additional tag so it dose not clash with anything else clients might be doing

Reply:
@hzrd149 I talked with @jb55 about this. And left the reasoning in the original post of this PR.

@jb55 might be able to comment further here.

But one key question I would have is what is the purpose of having a relay URL in a mute list? It makes sense in the context of notes. For mute lists however the relay URL is irrelevant. The user should be muted across all relay URLs.

Reply:
I agree, I think tag items can have different meaning depending on the context -- but maybe our heads will explode if we do that, not sure.

Reply:
> I agree, I think tag items can have different meaning depending on the context -- but maybe our heads will explode if we do that, not sure.

I think the earlier we shake the idea that we have to use the same parameters across each kind type the better. tag params for ubiquitous things like `p` will inevitably have different contextual parameters across many different kinds. I doubt we need pet names in mute lists or all future lists.

I noticed there's a table in the nips repo that tries to document these parameters but they are not scoped by kind, so that will be a large source of confusion going forward. contextual tag params should be described by the NIP describing the kind like this one.